config/src/config/indexer_grpc_config.rs

```diff
@@ -23,3 +32,36 @@ pub struct IndexerGrpcConfig {
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub output_batch_size: Option<u16>,
 }
+
```

Comment:
why not `impl Default for Indexer GrpcConfig {...}` ? It could set enabled=false and the other values. Then it's more clear what someone gets if they edit the .yaml and set enabled=true

Reply:
Yeah, so this is something you can argue with @bowenyang007 and @larry-aptos about 😄 I just moved things around. I agree -- I much prefer defaulting, too.